lascli/parser/secrets.py

```diff
@@ -1,26 +1,38 @@
+import json
+import pathlib
+
 from las import Client
 
 from lascli.util import nullable, NotProvided
 
 
+def _create_secret_dict(secret_data, secret_path):
```

Comment:
this seems like a general function that should be used for documents ground-truth as well.